sdk/keyvault/azure-security-keyvault-keys/test/ut/mocked_transport_adapter_test.hpp

```diff
@@ -9,20 +9,21 @@
 
 #include <gtest/gtest.h>
 
+#include "./../../src/private/key_serializers.hpp"
```

Comment:
```suggestion
#include <private/key_serializers.hpp>
```
This should work; we avoid relative paths in includes.